courses/views_test.py

```diff
@@ -194,20 +195,28 @@ def test_course_catalog_view(client):
     """
     program = ProgramFactory.create(live=True)
 
-    course_in_program = CourseFactory.create(program=program, live=True)
-    course_no_program = CourseFactory.create(no_program=True, live=True)
-
     # Required because catalog now excludes Programs and Courses that do not have a page
-    ProgramPageFactory.create(program=program)
-    CoursePageFactory.create(course=course_in_program)
-    CoursePageFactory.create(course=course_no_program)
+    program_page = ProgramPageFactory.create(program=program)
+    course_page_in_program = CoursePageFactory.create(
+        course=CourseFactory.create(program=program, live=True)
+    )
+    course_page_no_program = CoursePageFactory.create(
+        course=CourseFactory.create(no_program=True, live=True)
```

Comment:
This is a good change, but I think the `create` calls can be simplified:

```python
   course_page_in_program = CoursePageFactory.create(
      course__program=program, course__live=True
   )
   course_page_no_program = CoursePageFactory.create(
       course__no_program=True, course__live=True
   )
```

Reply:
Updated.